qgreenland/config/helpers/steps/compress_and_add_overviews.py

```diff
@@ -31,13 +38,20 @@ def compress_and_add_overviews(
 
             https://gdal.org/drivers/raster/gtiff.html
     """
-    predictor_value = 3 if dtype_is_float else 2
+    compress_creation_options = [
+        '-co', 'TILED=yes',
```

Comment:
Nitpick, make this `TILED=YES`? This is how the `gdal_translate` docs suggest setting the option

Reply:
a618192